

On x86_64:

drivers/isdn/hardware/eicon/debug.c: In function `DI_format':
drivers/isdn/hardware/eicon/debug.c:599: incompatible types in assignment

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-x86_64-akpm/drivers/isdn/hardware/eicon/debug.c |    5 +----
 1 files changed, 1 insertion(+), 4 deletions(-)

diff -puN drivers/isdn/hardware/eicon/debug.c~isdn-build-fix drivers/isdn/hardware/eicon/debug.c
--- 25-x86_64/drivers/isdn/hardware/eicon/debug.c~isdn-build-fix	Mon Aug 30 15:42:36 2004
+++ 25-x86_64-akpm/drivers/isdn/hardware/eicon/debug.c	Mon Aug 30 15:43:25 2004
@@ -573,7 +573,7 @@ static void DI_format (int do_lock,
                        unsigned short id,
                        int type,
                        char *format,
-                       va_list argument_list) {
+                       va_list ap) {
   diva_os_spin_lock_magic_t old_irql;
   dword sec, usec;
   diva_dbg_entry_head_t* pmsg = NULL;
@@ -582,7 +582,6 @@ static void DI_format (int do_lock,
   static char fmtBuf[MSG_FRAME_MAX_SIZE+sizeof(*pmsg)+1];
   char          *data;
   unsigned short code;
-  va_list ap;
 
   if (diva_os_in_irq()) {
     dbg_sequence++;
@@ -596,8 +595,6 @@ static void DI_format (int do_lock,
 
 
   
-  ap = argument_list;
-
   diva_os_get_time (&sec, &usec);
 
   if (do_lock) {
_
